policies/secure-development-lifecycle.md

```diff
@@ -0,0 +1,47 @@
+# Secure Development Lifecycle Policy
+
+## Purpose
+The secure developmet lifecycle describes the approach that is used to define, develop, deliver and maintain software. 
+
+## Scope
+This lifecycle is intended to be used by software teams including product managers, designers, developers and quality assurance to develop secure software products.
+
+## Policy
+This lifecycle integrates secruity into all aspects of application software development which is comprized of six phases. 
```

Comment:
```suggestion
This lifecycle integrates security into all aspects of application software development which is comprized of six phases. 
```

Reply:
Also don't forget: comprized -> comprised

Reply:
The "which" in "which is comprised of six phases" sounds like it's referring to "software development" but really refers to the "lifecycle". What do you think about:

``` suggestion
This lifecycle integrates security into all aspects of application software development and is comprised of six phases. 
```

or:

``` suggestion
This lifecycle, which aims to integrate security into all aspects of application software development, is comprised of six phases. 
```